include/eve/arch/riscv/rvv_common_masks.hpp

```diff
@@ -0,0 +1,54 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+namespace eve::detail
+{
+template<arithmetic_scalar_value T, typename N>
+EVE_FORCEINLINE logical<wide<T, N>>
+                rvv_true()
+{
+  static constexpr auto   lmul  = riscv_::getLMUL<T>(N::value);
+  static constexpr size_t size  = sizeof(T) * 8;
+  static constexpr size_t ratio = lmul > 0 ? size / lmul : size * (-lmul);
```

Comment:
Same, we shouldn't redo them over and over. There shuld be a 

```
template <simd_value T>
constexpr std::ptrdiff_t rvv_logical_ratio_v = ...
```

or smth

I'm also not sure what raio is. why ratio?

Reply:
Earlier it was something like `bit_size` (I earlier review I was asked to rename this). In terms of mask intrinsic types, this is just the number in type. E.g. `vbool64_t` - we get 64. I get it as `SEW/LMUL`. This could represent the number of bits that this mask is stores (you get the number of bits by `vlen/number`).

Also in rvv intrinsics documentation they use ratio too: `Mask types have the ratio that is derived ...`

For now named function `getLogicalRatio` and placed it in tags.